chrome/README.md

```diff
@@ -2,11 +2,12 @@
 
 To use the development version:
 
-1. Clone the repo `git clone https://github.com/RetireJS/retire.js`
-2. Run build_chrome.sh from the root of the repo
-3. Open [chrome://extensions/](chrome://extensions/)
-4. Check "Developer mode"
-5. Click "Load unpacked extension..." and locate the "chrome/extension" folder inside the repo
-6. Use and develop
+1. Clone the repo: `git clone https://github.com/RetireJS/retire.js`
+2. Install the requirements: `sudo apt install npm`
+3. Run `./build_chrome.sh` from the root of the repo
```

Comment:
Actually this line should originally have included "(or build_chome.bat if on Windows)"